File: week-03/day-01/30.py
ac = 8

time = 120

out = ''

if ac%4 is 0 and time < 200:
    out = "Check"
    print(out)
elif time > 200:
    out = "Time out"
    print(out)
else:
    out = "Run Forest Run!"
    print(out)

File: week-07/day-01/30.js
'use strict';

var ac = 8;
var time = 120;
var out = '';

if (ac % 4 === 0 && time <= 200) {
  console.log(out = 'Check');
} else if (time > 200) {
  console.log(out = 'Time out');
} else {
  console.log(out = 'Run Forest, run!');
}
// if ac is dividable by 4
// and time is not more than 200
// set out to 'check'
// if time is more than 200
// set out to 'Time out'
// otherwise set out to 'Run Forest Run!'
